dbms/src/Storages/DeltaMerge/Remote/DisaggTaskId.h

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <Flash/Mpp/MPPTaskId.h>
+#include <fmt/format.h>
+#include <kvproto/disaggregated.pb.h>
+
+namespace DB::DM
+{
+class DisaggTaskId
+{
+public:
+    static const DisaggTaskId unknown_disaggregated_task_id;
+
+public:
+    DisaggTaskId()
+        : DisaggTaskId(MPPTaskId::unknown_mpp_task_id, "")
+    {}
+
+    DisaggTaskId(MPPTaskId task_id, String executor_id_)
+        : mpp_task_id(std::move(task_id))
+        , executor_id(std::move(executor_id_))
+    {
+    }
+
+    explicit DisaggTaskId(const disaggregated::DisaggTaskMeta & task_meta);
+
+    disaggregated::DisaggTaskMeta toMeta() const;
+
+    const MPPTaskId mpp_task_id;
+    const String executor_id;
```

Comment:
Why `mpp_task_id` is not enough to identify? Because i think only `table scan` in RN will call `EstablishDisagTask()`, and there will only be one `table scan` in one MPPTask.

Reply:
There could be more than one `table scan` in one `MPPTask` (in the future)